src/puppet-js-view-model-tree.html

```diff
@@ -91,6 +91,10 @@
       var puppet = listener.getPuppetClient();
       if (puppet) {
         clone.querySelector('juicy-jsoneditor').json = puppet.obj;
+        // passover changes to Puppet
+        clone.querySelector('juicy-jsoneditor').addEventListener('change', function(e) {
+          puppet.puppet.handleLocalChange(e.detail.patches);
```

Comment:
I tried this in the beginning but they go unnoticed even after I clicked on the body. And I didn't investigate it further because it also needed adding jsonpatch external, it doesn't exist here.